HaploBlocker/haplonetwork.py

```diff
@@ -0,0 +1,316 @@
+from typing import List
+import numpy as np
+from collections import defaultdict
+from copy import copy
+
+BLOCK_SIZE = 20
+FILTER_THRESHOLD = 4
+
+
+def first(iterable):
+    return next(iter(iterable))
+
+
+class Point:
+    def __init__(self, snp, bp=0):
+        self.snp, self.bp = snp, bp
+
+    @property
+    def window(self):
+        return self.snp // BLOCK_SIZE
+
+
+class Node:
+    def __init__(self, ident, start, end, specimens=None, upstream=None, downstream=None):
+        self.ident = ident
+        self.start = start #Point()
```

Comment:
Yes these are SNP-numbers.
This originated from us using a SNP-chip data set to build the original graph. The only time they are actually used is for generating nodeIDs and to make it easier for us to check the results.

Do we need some kind of position for the visualization? BP-position might be different based on specimen or even multiple values for a specimens in case of CNV.